goagen/gen_controller/generator.go

```diff
@@ -108,6 +112,30 @@ func (g *Generator) Generate() (_ []string, err error) {
 	return g.genfiles, err
 }
 
+func (g *Generator) smartenPkg() {
+	if g.Pkg != "main" {
+		return
+	}
+	pwd, err := filepath.Abs(".")
```

Comment:
It's more clear. I'v replaced it. https://github.com/goadesign/goa/pull/1272/commits/a5e82032c8e582d704cf80c54fac77b1c318766f